src/components/20-molecules/file-upload/index.js

```diff
@@ -208,42 +208,48 @@ class AXAFileUpload extends LitElement {
     this.requestUpdate();
   }
 
+  validateOverallSize(fileSize = 0) {
+    const {
+      sizeOfAllFilesInBytes,
+      maxSizeOfAllFilesKB,
```

Comment:
`maxSizeIfAllFilesInKB`? For consistency I would also ad the `In` here

Reply:
OK, makes sense